src/Surfnet/StepupRa/RaBundle/Security/Authentication/Token/SamlToken.php

```diff
@@ -61,12 +79,12 @@ public function getLoa()
 
     public function serialize()
     {
-        return serialize([parent::serialize(), $this->loa,]);
+        return serialize([parent::serialize(), $this->loa, $this->institutionConfigurationOptions]);
```

Comment:
did you verify that the object (de)serializes correctly? It should, but still...